policy_definitions/tagging/policy_set_definition_qby_require_tags.tmpl.json

```diff
@@ -0,0 +1,39 @@
+{
+    "name": "QBY-Require-Tags",
+    "type": "Microsoft.Authorization/policySetDefinitions",
+    "apiVersion": "2021-06-01",
+    "scope": null,
+    "properties": {
+      "policyType": "Custom",
+      "displayName": "Require defined tags on resources",
+      "description": "Require resources and resource groups to have conceptual required tags.",
+      "metadata": {
+        "version": "1.0.0",
+        "category": "Tags"
+      },
+      "parameters": {},
+      "policyDefinitions": ${jsonencode(flatten([for tag_name in tags: [
```

Comment:
As you are using `tags`, I think now you would require any tag, that is inherited. There may be tags, that should be inherited, but should not be required.

Reply:
As now, we are ussing a custom policy definition, we don't have this files.